spec/core/cache_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "spec_helper"
+require "securerandom"
+
+module Fog
+  class SubFogTestModel < Fog::Model
+    identity  :id
+  end
+end
+
+module Fog
+  class SubFogTestService < Fog::Service
+
+    recognizes :region
+    class Mock
+      attr_reader :options
+      attr_accessor :region
+
+      def initialize(opts = {})
+        @options = opts
+      end
+    end
+  end
+end
+
+describe Fog::Cache do
+  it "will raise if no cache data found" do
+    Fog.mock!
+    service = Fog::SubFogTestService.new(:region => "us-east-1")
+    Fog::Cache.expire_cache!(Fog::SubFogTestModel, service)
```

Comment:
Question: Why do we care if there is no cache and we expire it? Or could their be an `expire_cache` method that doesn't raise?